From 3cbef2195533f357c8a80c2840108662461273b7 Mon Sep 17 00:00:00 2001
From: Lionel Orry <lionel.orry@gmail.com>
Date: Wed, 27 Mar 2013 14:48:19 +0100
Subject: [PATCH 4/4] Do not run tests.

This patch is specific to cross-compiled environments and avoids running
the tests on the host. It is not meant to be applied upstream.

Signed-off-by: Lionel Orry <lionel.orry@gmail.com>
---
 Makefile            | 2 +-
 tools/m2sh/Makefile | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 7dc4089..ca72630 100644
--- a/Makefile
+++ b/Makefile
@@ -16,7 +16,7 @@ TEST_SRC=$(wildcard tests/*_tests.c)
 TESTS=$(patsubst %.c,%,${TEST_SRC})
 MAKEOPTS=OPTFLAGS="${NOEXTCFLAGS} ${OPTFLAGS}" OPTLIBS="${OPTLIBS}" LIBS="${LIBS}" DESTDIR="${DESTDIR}" PREFIX="${PREFIX}"
 
-all: bin/mongrel2 tests m2sh procer
+all: bin/mongrel2 m2sh procer filters config_modules
 
 dev: CFLAGS=-g -Wall -Isrc -Wall -Wextra $(OPTFLAGS) -D_FILE_OFFSET_BITS=64
 dev: all
diff --git a/tools/m2sh/Makefile b/tools/m2sh/Makefile
index b50d8a0..334e4da 100644
--- a/tools/m2sh/Makefile
+++ b/tools/m2sh/Makefile
@@ -9,7 +9,7 @@ TESTS=$(patsubst %.c,%,${TEST_SRC})
 LIB_SRC=$(filter-out src/m2sh.c,${SOURCES})
 LIB_OBJ=$(filter-out src/m2sh.o,${OBJECTS})
 
-all: ../lemon/lemon tests build/m2sh
+all: ../lemon/lemon build/m2sh
 
 dev: CFLAGS=-g -Wall -Wextra -Isrc -I../../src $(OPTFLAGS)
 dev: all
-- 
1.8.1.4

